big_scape/cli/cluster_cli.py

```diff
@@ -35,7 +35,17 @@
     ),
 )
 # binning parameters
-@click.option("--no_mix", is_flag=True, help=("Don't run the all-vs-all analysis."))
+@click.option(
+    "--mix",
+    is_flag=True,
+    help=(
+        "Calculate distances using a 'mix' bin, wherein no classification is applied. "
+        "This will do an all-vs-all comparison, and is likely going to take a long time. "
+        "This bin will use weights from the 'mix' weights distribution. Warning: these "
+        "weights are not recommended for use with the record types protocluster/protocore, "
+        "as they have been optimized and validated only for the 'region' record type."
+    ),
+)
 @click.option(
```

Comment:
should legacy just go into the classify options? otherwise to run legacy classify one must do --classify none and --legacy_classify which is counter intuitive.

also make sure to adjust validations for legacy (if legacy_classify is on then legacy_weights also is)